psl-clj/src/psl_clj/core.clj

```diff
@@ -0,0 +1,347 @@
+(ns psl-clj.core
+  "A clojure user interface for PSL"
+  (:require
+   [clj-util.core :as u]
+   [clojure.tools.logging :as log]
+   [incanter.core :as in]
+   [psl-clj.Model]
+   )
+  (:import
+   [psl_clj Model]
+   [org.linqs.psl.application.inference MPEInference LazyMPEInference]
+   [org.linqs.psl.application.util GroundRules Grounding]
+   [org.linqs.psl.database DataStore Database Partition]
+   [org.linqs.psl.model.term UniqueStringID Variable Term Constant]
+   [org.linqs.psl.database.rdbms RDBMSDataStore]
+   [org.linqs.psl.model.atom QueryAtom]
+   [org.linqs.psl.model.formula Conjunction Disjunction Negation Formula Implication]
+   [org.linqs.psl.model.rule.logical UnweightedGroundLogicalRule
+    UnweightedLogicalRule WeightedLogicalRule]
+   [org.linqs.psl.database Queries]
+   [java.util HashSet]
+   ))
+
+;;; ================== Building PSL models ===========================
```

Comment:
Would it be possible to use the psl-parser?
Then you don't have to worry about any rule building.

Reply:
That could be beneficial so that the full features of psl-parser are available.  Building rules using the flexibility of the Clojure language, however, is one of main advantages of this interface, so I recommend against a full switch from one to the other.

Reply:
I added psl-parser like you said.  See PR2.